emukit/quadrature/interfaces/base_gp.py

```diff
@@ -49,6 +49,7 @@ def gram_chol(self) -> np.ndarray:
 
         :return: a lower triangular cholesky of G(X, X)
         """
+        # Todo: is it too much to impose the cholesky? maybe inverse would be better
```

Comment:
Can you raise an issue explaining this TODO in more detail, in case someone wants to pick it up?

Reply:
This is related to the discussion we had about the BQ interface. We can discuss offline.